cli/src/main/resources/application.yaml

```diff
@@ -47,6 +47,9 @@ sbomer:
     api-url: "${SBOMER_CACHE_SERVICE_URL:${PNC_SERVICE_HOST:orch.psi.redhat.com}}"
     product-mapping:
       environment: "${SBOMER_PRODUCT_MAPPING_ENV:prod}"
+  koji:
+    hub-url: "https://brewhub.engineering.redhat.com/brewhub"
+    web-url: "https://brewweb.engineering.redhat.com/brew"
```

Comment:
The configuration refactoring unified the way we define server addresses. I based on using the `sbomer.[NAME].host` pattern. In this case it would be: `sbomer.koji.host`.

So, I would set it to followng:

```yaml
sbomer:
  koji:
    hub-url: "https://${sbomer.koji.host}/brewhub"
    web-url: "https://${sbomer.koji.host}/brew"
```

Afterwards, in the `task-generate.yaml` file I would add following env variable:

```yaml
- name: SBOMER_KOJI_HOST
  value: {{ .Values.koji.host }}
```

And finally, add the hostname to the sbomer-support repo we have, for all environments.

```yaml
koji:
  host: "THEHOSTNAME"
```

(Actually, I've done the last part for ya already, a MR is open)

P.S This of course implies that you rebase 😀